src/vonage/messages.py

```diff
@@ -2,31 +2,32 @@
 
 import re
 
+
```

Comment:
Do you want to pull this out so you can merge in without conflicts?

Reply:
Yeah I'll do this